app/views/welcome/index.html.erb

```diff
@@ -4,6 +4,13 @@
 <div class="col-md-6 col-md-offset-3 text-center">
     <h3>Future Home Of the Fall Gambit Chess App!</h3>
     <br />
-    <%= render "available_games" %>
+    <% if @games.present? %>
```

Comment:
One is checking if there are available games, one if there are games at all. If we don't have the first if-statement and there is no game in the db `.each` is called on NilClass. But yes, there's probably a better way to do it.